resources/home/dnanexus/eggd_artemis.py

```
@@ -2,178 +1,29 @@
+"""Main app entrypoint for the app"""
```

Ruff (0.8.0)

The file is executable but no shebang is present

(EXE002)